### PR TITLE
New product details component 

### PR DESCRIPTION
**Pull request by charleycampbell**
## What does this change?
Rob has created a new product details table which has the subscription id and start date on it. It is a different design to the existing component and has been added to the manage products V2.

[Design](https://www.figma.com/file/sx9jlOcpcK9V0gNgj4GWtW/Cancellation-saves-Q3-2023?node-id=543%3A5553&mode=dev) on Figma

## How to test
Check new storybook component test and manage product V2 storybook

## Images

image

image

## Accessibility

-   [ ] [Tested with screen reader](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#screen-reader)
-   [ ] [Navigable with keyboard](https://github.com/guardian/accessibility/blob/main/people-and-technology/02-physical.md#keyboard)
-   [ ] [Colour contrast passed](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#contrast)
-   [ ] [The change doesn't use only colour to convey meaning](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#use-of-colour)

### CODE REVIEW

File: client/components/mma/shared/ProductInfoTableV2.tsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+export const ProductInfoTableV2 = (props: ProductInfoTableProps) => {
+	const valueBoxCss = () => {
+		return css`
+			background-color: ${palette.neutral[97]};
+			display: flex;
+			width: 803px;
+			${until.tablet} {
+				width: 351px;
+			}
+			${until.tablet} {
```
Comment (andrewHEguardian) [inline client/components/mma/shared/ProductInfoTableV2.tsx:25]: looks like this snuck in

File: client/components/mma/shared/ProductInfoTableV2.tsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+}
+
+export const ProductInfoTableV2 = (props: ProductInfoTableProps) => {
+	const valueBoxCss = () => {
+		return css`
+			background-color: ${palette.neutral[97]};
+			display: flex;
+			width: 803px;
+			${until.tablet} {
+				width: 351px;
```
Comment (andrewHEguardian) [inline client/components/mma/shared/ProductInfoTableV2.tsx:23]: not sure about these widths, feel like there may be a nicer way to do it

File: client/components/mma/shared/ProductInfoTableV2.tsx [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+		`;
+	};
+
+	const tableValueCss = () => {
+		return css`
+			color: #606060;
+			padding-right: 32px;
+			width: 100%;
+			${from.tablet} {
+				width: auto;
```
Comment (andrewHEguardian) [inline client/components/mma/shared/ProductInfoTableV2.tsx:43]: just trying to understand, what do the different widths do at each screen size?

File: client/components/mma/shared/ProductInfoTableV2.tsx (hunk 2 above)
Comment (andrewHEguardian) [reply client/components/mma/shared/ProductInfoTableV2.tsx:23]: image
is the width going past the edge?

**Review by andrewHEguardian — DISMISSED** [review]
minor comment about the width but otherwise looks really good 🖌️ 🎨  love the new design!

File: client/components/mma/shared/ProductInfoTableV2.tsx (hunk 3 above)
Comment (charleycampbell) [reply client/components/mma/shared/ProductInfoTableV2.tsx:43]: This isn't required so I've removed it

File: client/components/mma/shared/ProductInfoTableV2.tsx [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+`;
+
+const tableEntryCss = css`
+	${textSans.medium()};
+	margin-right: ${space[2]}px;
+`;
+
+const tableValueCss = css`
+	color: #606060;
+	${palette.neutral[38]}
```
Comment (andrewHEguardian) [inline client/components/mma/shared/ProductInfoTableV2.tsx:35]:
```suggestion
```

**Review by andrewHEguardian — APPROVED** [review]
👏 👍

**Pull request merged**